packages/db/src/translate/pdrs.ts

```diff
@@ -0,0 +1,44 @@
+import Knex from 'knex';
+
+import { ApiPdr } from '@cumulus/types/api/pdrs';
+
+import { CollectionPgModel } from '../models/collection';
+import { ExecutionPgModel } from '../models/execution';
+import { ProviderPgModel } from '../models/provider';
+import { PostgresPdr } from '../types/pdr';
+
+export const translatePostgresPdrToApiPdr = async (
+  postgresPDR: PostgresPdr,
+  knex: Knex | Knex.Transaction,
+  collectionPgModel = new CollectionPgModel(),
+  executionPgModel = new ExecutionPgModel(),
+  providerPgModel = new ProviderPgModel()
+): Promise<ApiPdr> => {
+  const collection = await collectionPgModel.get(knex, {
+    cumulus_id: postgresPDR.collection_cumulus_id,
+  });
+  const execution = await executionPgModel.get(knex, {
+    cumulus_id: postgresPDR.execution_cumulus_id,
+  });
+  const provider = await providerPgModel.get(knex, {
+    cumulus_id: postgresPDR.provider_cumulus_id,
+  });
+
+  return {
```

Comment:
Do we want/need to omit nil properties from the return? What does the API return if some optional field like `address` comes back from this function as `undefined`?

Reply:
I think generally we should be, as that's the expected behavior in other endpoints (e.g. providers).    Having `undefined` coming back can break schemas in some cases, etc.

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/8ba0c6ae4c1b80cdfd5e79079bae29ddcd64d26b